saleor/graphql/account/i18n.py

```diff
@@ -5,8 +5,43 @@
 from ...account.forms import get_address_form
 from ...account.models import Address
 from ...account.validators import validate_possible_number
+from ...core.exceptions import PermissionDenied
+from ...permission.auth_filters import AuthorizationFilters
+from ...permission.enums import (
+    AccountPermissions,
+    BasePermissionEnum,
+    CheckoutPermissions,
+    OrderPermissions,
+    ProductPermissions,
+    SitePermissions,
+)
+from ...permission.utils import all_permissions_required
 from ..core import ResolveInfo
 
+SKIP_ADDRESS_VALIDATION_PERMISSION_MAP: dict[str, list[BasePermissionEnum]] = {
+    "addressCreate": [AccountPermissions.MANAGE_USERS],
+    "addressUpdate": [AccountPermissions.MANAGE_USERS],
+    "draftOrderCreate": [OrderPermissions.MANAGE_ORDERS],
+    "draftOrderUpdate": [OrderPermissions.MANAGE_ORDERS],
+    "orderUpdate": [OrderPermissions.MANAGE_ORDERS],
+    "orderBulkCreate": [OrderPermissions.MANAGE_ORDERS_IMPORT],
+    "createWarehouse": [ProductPermissions.MANAGE_PRODUCTS],
+    "updateWarehouse": [ProductPermissions.MANAGE_PRODUCTS],
+    "shopAddressUpdate": [SitePermissions.MANAGE_SETTINGS],
+    "checkoutCreate": [
+        CheckoutPermissions.HANDLE_CHECKOUTS,
+        AuthorizationFilters.AUTHENTICATED_APP,
+    ],
+    "checkoutShippingAddressUpdate": [
+        CheckoutPermissions.HANDLE_CHECKOUTS,
+        AuthorizationFilters.AUTHENTICATED_APP,
+    ],
+    "checkoutBillingAddressUpdate": [
+        CheckoutPermissions.HANDLE_CHECKOUTS,
+        AuthorizationFilters.AUTHENTICATED_APP,
+    ],
```

Comment:
Should we add information about required permissions on those mutations when skipping validation?  
e.g. on `createWarehouse` we have information about the required permission. On the `checkoutCreate` there is no such information. 

![image](https://github.com/saleor/saleor/assets/43955230/469ebed3-a32d-4582-9ade-b22983defcd8)
![image](https://github.com/saleor/saleor/assets/43955230/b3b7fff2-779f-4aa0-9fe1-ea7bc9e3f3f5)

Reply:
Thanks for comment. I am not sure if we should change the description of whole `checkoutCreate` mutation, just to  describe behaviour of single flag. Maybe we should somehow extend the description of `AddressInput.skip_validation` field? Any idea how it should looks like?

Reply:
I don't have any idea right now. I'm not sure about adding this info to `AddressInput` other mutations have complete information about required permission in their own description not in input descriptions.

Reply:
The info added to `createMutation` description.